tools/static-assets/skel-apollo/imports/ui/Info.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useQuery } from '@apollo/react-hooks';
```

Comment:
I believe you need to change this as well

```suggestion
import { useQuery } from '@apollo/client';
```